src/script/auth/page/Login.tsx

```diff
@@ -296,9 +299,11 @@ const Login = ({
                         data-uie-name="enter-code"
                         markInvalid={!!twoFactorSubmitError}
                       />
-                      {!!twoFactorSubmitError && parseError(twoFactorSubmitError)}
                     </Label>
-                    <div style={{marginTop: 30}}>
+                    <div style={{display: 'flex', justifyContent: 'center', marginTop: 10}}>
```

Comment:
@tlebon why are we adding inline styles not a class?

Reply:
Don't we use emotion? Why are we adding classes when using css-in-js? :D

Reply:
i mean i didnt change the method of styling here.. just extended it to fix a small bug